### ISSUE TITLE
Request - Need a Dimensional Blacklist 1.10.2

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [setycz/ChickensMod#78]
#### Issue Description:
The issue is sometimes chickens and more chickens are spawning in undesirable places.

**Comment by Kazzamkablam** [comment]
yep, happens with compact machines, if you got any grass blocks inside random chickens will spawn turning it to a mess.

**Comment by GenDeathrow** [comment]
@Kazzamkablam I believe this can be done with a mod called In-Control I believe. I will still look into adding this later.
